Source/Urho3D/Scene/Node.h

```diff
@@ -801,6 +826,8 @@ class URHO3D_API Node : public Serializable
     Node* GetChildByNameOrIndex(ea::string_view name, bool recursive = false) const;
     /// Find component by name. If name is empty, returns the owner node itself.
     Serializable* GetSerializableByName(ea::string_view name) const;
+    /// Find components. Returns true to continue or false if search is over.
+    template <typename Callback> bool FindComponents(ComponentSearchFlags flags, StringHash typeId, Callback callback) const;
```

Comment:
I think we should do const& callback because it is passed into this function a lot of times